src/core/ui/components/drop-in/drop-down.tsx

```diff
@@ -0,0 +1,414 @@
+import { useUser } from '../../context/user';
+import { useMemo, useState } from 'react';
+import { useSlashAuth } from '../../../context/legacy-slashauth';
+import { ScaleLoader } from 'react-spinners';
+
+export const DropDown = () => {
+  const user = useUser();
+  const [isOpen, setIsOpen] = useState(false);
+  const context = useSlashAuth();
+  const {
```

Comment:
Let's not use `useSlashAuth` here. That's an external hook and we should be using internal hooks within our components (as they are wrapped with specific contexts.)

For example, if you want to open the login modal, you can use `useCoreSlashAuth()` to extract the `openSignIn` flow

Reply:
updated